packages/strapi/lib/Strapi.js

```diff
@@ -249,9 +249,33 @@ class Strapi {
         }
       }
 
-      // Emit started event.
+      // Get database clients
       const databaseClients = _.map(this.config.get('connections'), _.property('settings.client'));
-      await this.telemetry.send('didStartServer', { database: databaseClients });
+
+      // Get plugins
+      const plugins = { strapi: [], external: [] };
+      for (const plugin in this.plugins) {
+        if (this.plugins[plugin].package.author.name === 'Strapi team') {
+          plugins.strapi.push(this.plugins[plugin].package.name);
+        } else {
+          plugins.external.push(this.plugins[plugin].package.name);
+        }
+      }
+
+      // Get providers
+      const providers = _(this.plugins)
```

Comment:
You should check the strapi-provider-xxx pacakges in strapi package json not for each plugin as this won't find manually installed providers. 

* Avoid using lodash chaining api